rt_entt_codegen/core/system_provider/association/association.cc

```diff
@@ -0,0 +1,335 @@
+#include "association.hh"
+
+#include <map>
+
+#include "ecsact/lang-support/lang-cc.hh"
+#include "rt_entt_codegen/shared/system_util.hh"
+#include "rt_entt_codegen/shared/util.hh"
+#include "ecsact/runtime/meta.hh"
+#include "ecsact/cpp_codegen_plugin_util.hh"
+
+using capability_t =
+	std::unordered_map<ecsact_component_like_id, ecsact_system_capability>;
+
+auto ecsact::rt_entt_codegen::core::provider::association::
```

Comment:
some of these are horrendeously long. maybe add a `using namespace ecsact::rt_entt_codegen::core` so these become 
```suggestion
auto provider::association::
```